coffee-shop/style.css

```diff
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+* {
+    margin: 0px ;
+    padding: 0px ;
+}
+body {
+    background-color: blanchedalmond;
+}
+nav {
+   height: 100px;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   flex-direction: column;
+   color: rgb(254, 90, 61);
+}
+nav > h1 {
+    text-transform: uppercase;
+    font-size: 50px;
+}
+.background-1 {
+    background-color: rgb(254, 90, 61);
```

Comment:
Do you like bloody colors?)

Reply:
hehe just found a nice colour scheme on Pinterest ;)